README.md

```diff
@@ -225,6 +225,18 @@ Log types are exposed as `consola.[type](...)` and each is a preset of styles an
 
 A list of all available built-in types is [available here](./src/constants.ts).
 
+## Raw output
+
+Objects sent to reporter by logging them could be an issue when using conflicting key (esp. `args`).
```

Comment:
We might want to add a link to [`isLogObj` function](https://github.com/unjs/consola/blob/ce2b368f43813aa95e54d6124425b1c6badc087e/src/utils/log.ts#L5), or add more description about conflicting keys, because conflicting keys are not so many, just a few.

Or, we could cover it in examples, as https://github.com/unjs/consola/pull/271#issuecomment-1882790447

Reply:
I like the idea to add a link to the isLogObj and we could add a note that this is an advanced feature.